src/styles/style.css

```diff
@@ -503,4 +503,10 @@ table tbody td {
 #mc header,
 section {
   margin-bottom: 6rem;
+}
+
+.comp-show {
+  background-color: #E8E8E8;
+  padding: 30px;
+  margin-bottom: 20px;
```

Comment:
These two as well, please.
We don't have 30px spacing, only 20.8px and 41.6px so maybe we can do 20.8px for both?
The token is: `--gcds-spacing-40`